README.md

```diff
@@ -132,27 +134,29 @@ Possible values are:
 | Setting       | Values                                |
 |---------------|---------------------------------------|
 | Mode          | `PS1`, `PS2`                          |
-| AutoBoot      | `OFF`, `ON`                           |
-| GameID        | `OFF`, `ON`                           |
+| FlippedScreen | `ON`, `OFF`                           |
+| AutoBoot      | `ON`, `OFF`                           |
+| GameID        | `ON`, `OFF`                           |
 | CardSize      | `1`, `2`, `4`, `8`, `16`, `32`, `64`  |
 | Variant       | `RETAIL`, `PROTO`, `ARCADE`           |
-| FlippedScreen | `ON`, `OFF`                           |
+
+*Note: `Variant=ARCADE` is for Namco System 246/256 or Konami Python 1 arcade machines and only the first memory card slot is supported. `RETAIL` is for regular PS2 machines. `PROTO` is for unknown prototype models.*
```

Comment:
This is not complete 



(I was thinking of renaming the arcade mode to DONGLE)

RETAIL works with retail, DEX and arcade keys (first set of keys, arcade PS2 mc slot 2). Basically, RETAIL mode supports all the same mg variants than a SCPH-10020 excluding the unknown keys